doc/_static/mpl.css

```diff
@@ -1278,3 +1274,7 @@ div.bullet-box li {
         flex: 0 0 90%;
     }
 }
+
+div#gallery.section .sphx-glr-clear:first-of-type, div#tutorials.section .sphx-glr-clear:first-of-type{
+    display: none;
+}
```

Comment:
Do we want a newline at the end?

Reply:
I'm not super worried about that in the css files.